boilerplates/be-docker-plain/Jenkinsfile

```diff
@@ -0,0 +1,55 @@
+def final projectId = '@project_id@'
+def final componentId = '@component_id@'
+def final credentialsId = "${projectId}-cd-cd-user-with-password"
+def sharedLibraryRepository
+def dockerRegistry
+node {
+  sharedLibraryRepository = env.SHARED_LIBRARY_REPOSITORY
+  dockerRegistry = env.DOCKER_REGISTRY
+}
+
+library identifier: 'ods-library@0.1-latest', retriever: modernSCM(
+  [$class: 'GitSCMSource',
+   remote: sharedLibraryRepository,
+   credentialsId: credentialsId])
+
+// See readme of shared library for usage and customization.
+odsPipeline(
+  image: "${dockerRegistry}/cd/jenkins-slave-nodejs8-angular",
+  projectId: projectId,
+  componentId: componentId,
+  verbose: true
+) { context ->
+  stageBuild(context)
+  stageUnitTest(context)
+  /*
+   * if you want to introduce scanning, uncomment
+   * 
+   * stageScanForSonarqube(context)
+   */ 
+  stageCreateOpenshiftEnvironment(context)
+  stageStartOpenshiftBuild(context)
+  stageDeployToOpenshift(context)
+  stageTriggerAllBuilds(context)
```

Comment:
Both `stageCreateOpenshiftEnvironment(context)` and `stageTriggerAllBuilds(context)` have been removed in 1.0.0.

Reply:
@michaelsauter that osb2hy inpoint to 0.1 ;-)

Reply:
But you are merging to `master`, so you need to point to 1.0.0 and remove this :) On `master`, we should not point to an outdated, incompatible version ...

Reply:
@michaelsauter  I believe i leave this currently on 0.1 .. and when we tag 1.1 we change it, ok?